pkg/labels/labels.go

```diff
@@ -98,8 +99,9 @@ type Label struct {
 	Key   string `json:"key"`
 	Value string `json:"value,omitempty"`
 	// Source can be on of the values present in const.go (e.g.: CiliumLabelSource)
-	Source string `json:"source"`
-	absKey string
+	Source   string `json:"source"`
+	absKeyMU sync.RWMutex
```

Comment:
There was a race condition, the `AbsoluteKey()` is called in multiple place like policy check, that can happen in parallel.